pkg/controller/job/job_controller.go

```diff
@@ -824,7 +823,7 @@ func (jm *Controller) manageJob(job *batch.Job, activePods []*v1.Pod, succeeded
 			diff = 0
 		}
 		if diff == 0 {
-			return active, nil
+			return active, metrics.JobSyncActionTracking, nil
```

Comment:
how do the above two conditions ever be possible if wantActive is strictly larger than active in this branch?

Reply:
they are not possible, but the code was there 🤷‍♂️

Reply:
that said, let's remove it.